File: Algorithms/01-Warmup/07-min-max-sum/07.java
import java.util.Scanner;

class Solution {

    static void miniMaxSum(int[] arr) {
        long higher = 0;
        long lower = 0;
        for (int i = 0; i < arr.length; i++) {
            long sum = 0;

            for (int j = 0; j < arr.length; j++) {
                if (j != i) {
                    sum += arr[j];
                }
            }
            if (i == 0 || lower > sum) {
                lower = sum;
            }
            if (sum > higher) {
                higher = sum;
            }
        }
        System.out.printf("%d %d", lower, higher);
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int[] arr = new int[5];
        for(int arr_i = 0; arr_i < 5; arr_i++){
            arr[arr_i] = in.nextInt();
        }
        miniMaxSum(arr);
        in.close();
    }
}

File: Algorithms/01-Warmup/07-min-max-sum/07.js
process.stdin.resume();
process.stdin.setEncoding('ascii');

var input_stdin = "";
var input_stdin_array = "";
var input_currentline = 0;

process.stdin.on('data', function (data) {
    input_stdin += data;
});

process.stdin.on('end', function () {
    input_stdin_array = input_stdin.split("\n");
    main();
});

function readLine() {
    return input_stdin_array[input_currentline++];
}

/////////////// ignore above this line ////////////////////

function miniMaxSum(arr) {
    var higher = 0;
    var lower = 0;

    for (var i = 0; i < arr.length; i++) {
        var sum = 0;
        for (var j = 0; j < arr.length; j++) {
            if (j != i) {
                sum += arr[j];
            }
        }
        if (i === 0) {
            lower = sum;
        } else {
            if (lower > sum) {
                lower = sum;
            }
        }
        if (sum > higher) {
            higher = sum;
        }
    }
    console.log(lower + " " + higher);
}

function main() {
    var arr = readLine().split(' ');
    arr = arr.map(Number);
    miniMaxSum(arr);

}
